index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <title>Portofolio</title>
+  </head>
+  <body>
+    <header>
```

Comment:
Background image for the Headline section should position under the hamburger menu and it should not overflow on the hamburger menu. Kindly see the screenshot below and follow the design to place background images on the page.
![header-back](https://user-images.githubusercontent.com/35504149/133491154-5715e7d7-1431-4932-9945-df3dfe0e5f43.png)